File: guides/micronaut-microservices-distributed-tracing-zipkin/bookrecommendation/java/src/main/java/example/micronaut/Application.java
package example.micronaut;

import io.micronaut.context.env.Environment;
import io.micronaut.runtime.Micronaut;

public class Application {

    public static void main(String[] args) {
        Micronaut.build(args)
                .mainClass(Application.class)
                .defaultEnvironments(Environment.DEVELOPMENT)
                .start();
    }
}

File: guides/micronaut-microservices-distributed-tracing-jaeger-opentelemetry/bookcatalogue/kotlin/src/main/kotlin/example/micronaut/Application.kt
package example.micronaut

import io.micronaut.context.env.Environment
import io.micronaut.runtime.Micronaut

class Application {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            Micronaut.build(*args)
                    .mainClass(Application::class.java)
                    .defaultEnvironments(Environment.DEVELOPMENT)
                    .start()
        }
    }
}